package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-models",
-  "version": "0.10.1",
+  "version": "0.10.2",
```

Comment:
The version should be 0.10.3 . The current version in DEV branch is 0.10.2 (not sure how this diff was created , but something looks wrong)